guides/fields/introduction.md

```diff
@@ -87,16 +87,18 @@ By default, fields return values by:
 
 - Trying to call a method on the underlying object; _OR_
 - If the underlying object is a `Hash`, lookup a key in that hash.
-- An optional `:fallback_value` can be supplied that will be used if the above fail.
+- An optional `:fallback_value` can be supplied that will define a default value if the above fail or return `nil`.
```

Comment:
So my particular use case is/was where a we return counts.

I took 2 different approaches. One approach returned a hash, so this would be ok (the keys existed, but had `nil` values).

But the other approach did not.

Perhaps we do need to introduce something more specifically for `nil_value:`. Wdyt?